packages/fakta/fakta-omsorg-og-rett/src/components/OmsorgOgRettInfoPanel.tsx

```diff
@@ -29,23 +32,36 @@ export const OmsorgOgRettInfoPanel = ({ personoversikt, ytelsefordeling, submitt
         </AksjonspunktHelpTextHTML>
       )}
 
-      <PersonopplysningerForFamilie alleKodeverk={alleKodeverk} personoversikt={personoversikt} />
+      <OpplysningerFraSoknad omsorgOgRett={omsorgOgRett} alleKodeverk={alleKodeverk} />
+      {personoversikt.bruker?.adresser && (
+        <OpplysningerOmAdresser alleKodeverk={alleKodeverk} personoversikt={personoversikt} />
+      )}
+      {omsorgOgRett.registerdata && <AnnenPartsYtelser omsorgOgRett={omsorgOgRett} />}
 
       {harAPAleneomsorg && (
         <AleneomsorgForm
-          ytelsefordeling={ytelsefordeling}
+          omsorgOgRett={omsorgOgRett}
           submittable={submittable}
           aksjonspunkt={aksjonspunkterForPanel[0]}
         />
       )}
-
       {harAPAnnenForelderRett && (
         <HarAnnenForelderRettForm
-          ytelsefordeling={ytelsefordeling}
+          omsorgOgRett={omsorgOgRett}
           submittable={submittable}
           aksjonspunkt={aksjonspunkterForPanel[0]}
         />
       )}
+      {!harAPAleneomsorg &&
```

Comment:
Flytte ut denne til eigen variabel? (Ser ut som den er felles for begge sjekkane): 
```
!harAPAleneomsorg &&
        !harAPAnnenForelderRett &&
        omsorgOgRett.manuellBehandlingResultat
```